src/snowflake/cli/plugins/notebook/commands.py

```diff
@@ -49,3 +57,16 @@ def open_cmd(
     url = NotebookManager().get_url(notebook_name=identifier)
     typer.launch(url)
     return MessageResult(message=url)
+
+
+@app.command(requires_connection=True)
+def create(
+    identifier: Annotated[NotebookName, NOTEBOOK_IDENTIFIER],
+    notebook_file: Annotated[NotebookStagePath, NotebookFile],
+    **options,
+):
+    """Creates notebook from stage."""
+    results = NotebookManager().create(
+        notebook_name=identifier, notebook_file=notebook_file
+    )
+    return MultipleResults(QueryResult(c) for c in results)
```

Comment:
Can we try to return link to the notebook? Would it make sense?

Reply:
Sure.
Should we add link as next position to result or replace result with link?

Reply:
The results is meaningless for now, so we should do similar think as we do in streamlit deploy

Reply:
🫡